src/test/scala/org/hammerlab/guacamole/main/GeneratePartialFasta.scala

```diff
@@ -77,15 +80,6 @@ object GeneratePartialFasta extends Logging {
       })
     })
     writer.close()
-    Common.progress("Wrote: %s".format(args.output))
-  }
-
-  private def printUsage() = {
```

Comment:
Just noting that I think this is one of the worst limitations of args4j. Would really like to replace it with something that allows us to give a usage message to users for each tool (#106) .

Also thanks for making sure that `GeneratePartialFasta` continues to be runnable. Need to ensure that we can always run https://github.com/hammerlab/guacamole/blob/master/scripts/generate-partial-fasta-for-tests whenever we need to regenerate the partial fastas.